tests/workflows/basic_tool_calling_node/tests/test_workflow.py

```diff
@@ -542,6 +542,22 @@ def generate_prompt_events(*_args, **_kwargs) -> Iterator[ExecutePromptEvent]:
     # THEN the adhoc_execute_prompt_stream should be called exactly twice
     assert vellum_adhoc_prompt_client.adhoc_execute_prompt_stream.call_count == 2
 
+    # AND the second call should have the correct chat_history input value
+    second_call = vellum_adhoc_prompt_client.adhoc_execute_prompt_stream.call_args_list[1]
+    chat_history_input = next(
+        input_val
+        for input_val in second_call.kwargs["input_values"]
+        if hasattr(input_val, "key") and input_val.key == "chat_history"
+    )
+    assert chat_history_input.value == [
+        ChatMessage(
+            text="I'll help you get the weather information.",
+            role="ASSISTANT",
+            content=None,
+            source=None,
+        ),
+    ]
```

Comment:
great. Now instead of this shape, it should actually be a two element array on `content`. This will fail on main. Once you make that edit, update our implementation to make the test pass